src/store/module/task.module.js

```diff
@@ -0,0 +1,40 @@
+const task = {
+  nameSpaced: true,
+  state () {
+    return {
+      todos: []
+    }
+  },
+  mutations: {
+    addValues (state, payload) {
+      state.todos.push(payload)
+    },
+    delete (state, payload) {
+      state.todos.splice(payload, 1)
+    },
+    edit (state, payload) {
+      state.todos.splice(payload.index, 1, {
+        task: payload.task,
+        description: payload.description,
+        assignedTo: payload.assignedTo
```

Comment:
Instead of passing the edited content like this, You can directly give payload here. Since we are sending the edited item in this same structure itself. Just like **state.todos.splice(payload.index, 1, payload**